### DISCUSSION TITLE
which is much better to use? Docker vs containerd?

### DISCUSSION THREAD

**Discussion by vhuezo** [k3s-io/k3s#7828] [Q&A]
which is much better to use? Docker vs containerd?
`
curl -sfL https://get.k3s.io | sh -s - --docker`

vs

`curl -sfL https://get.k3s.io | sh -`

![imagen](https://github.com/k3s-io/k3s/assets/41808201/e786c2b0-3d90-45a4-8a22-5e933e206570)

**Comment by brandond** [comment]
Docker itself uses containerd as the runtime engine. Unless you have some compelling reason to use docker, I would recommend skipping the multiple additional layers of abstraction and just use containerd directly.
